apps/web/package.json

```diff
@@ -47,12 +47,12 @@
     "classnames": "2.5.1",
     "eslint": "^8",
     "eslint-config-custom": "workspace:*",
+    "postcss": "8.4.40",
+    "tailwindcss": "3.4.3",
```

Comment:
did this change position for some reason? can we leave it sorted? if we also remove the updated version from radix-ui/react-tooltip I believe this file would be unchanged and less things in the PR 😅

Reply:
Done!